nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/CandidateBO.java

```diff
@@ -115,10 +116,17 @@ public static Optional<CandidateBO> fromRequest(UpsertCandidateRequest request,
         return Optional.empty();
     }
 
+    @JacocoGenerated
+    public CandidateDao getCandidateDao() {
```

Comment:
Maybe it's better to create getters per value needed from candidateDao().cadidate()
f.ex
bool applicable(){
return candidateDao.candidate().applicable();
}

And same concept if you need a field from ApprovalStatusBO
candidateBo.approvalStatus(institutionId).username() 

Username username(){
return approvalStatusDao.approvalStatus().username();
}